File: Assignment/Module 3 New/SE- Fundamental of Programming/Vowel or Consonant.c
#include <stdio.h>
main() 
{
    char ch;

    printf("Enter a character: ");
    scanf(" %c", &ch);

    ch = tolower(ch);

    switch (ch) 
	{
        case 'a':
        case 'e':
        case 'i':
        case 'o':
        case 'u':
            printf("Vowel\n");
            break;
        default:
            printf("Consonant\n");
            break;
    }
}

File: Assignment/Module 1 Java Intoduction/VowelOrConsonant.java
package com.simple;

import java.util.Scanner;

public class VowelOrConsonant
{
	public static void main(String[] args)
	{
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter the Alphabet : ");
		String a=sc.next();
		char ch=a.charAt(0);
		
		if(a.length()>1)
		{
			System.out.println("Error..");
		}
		else if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u' || ch=='A' || ch=='E' || ch=='I' || ch=='O' || ch=='U')
		{
			System.out.println("Alphabet is Vowel..");
		}
		else
		{
			System.out.println("Alphabet is Consonant..");
		}
	}
}
